kinto/views/__init__.py

```diff
@@ -0,0 +1,26 @@
+import random
+import string
+
+from cliquet.storage import generators, exceptions
+from pyramid import httpexceptions
+
+
+class NameGenerator(generators.Generator):
+    def __call__(self):
+        try:
+            ascii_letters = string.ascii_letters
+        except ImportError:
+            ascii_letters = string.letters  # NOQA
+        alphabet = ascii_letters + string.digits + '-'
+        letters = [random.choice(alphabet) for x in range(8)]
+        return ''.join(letters)
```

Comment:
We add a nice thing in Daybed: https://github.com/spiral-project/daybed/blob/cb5efbf9b2ef326154a638c242d64ae001600ba4/daybed/backends/id_generators.py#L18-L41